apps/src/lab2/levelEditors/predictSettings/MultipleChoiceFields.tsx

```diff
@@ -28,7 +28,7 @@ const MultipleChoiceFields: React.FunctionComponent<
     }
     setPredictSettings({
       ...predictSettings,
-      multipleChoiceAnswers: newCorrectAnswers,
+      solution: newCorrectAnswers.join(','),
```

Comment:
Any concerns that if the order is changed for multiple choice answers, the solution might be checked incorrectly?

Reply:
We don't actually validate their answers anywhere. But we could avoid issues with ordering pretty easily by doing an includes rather than checking equality.